osm2pgsql.lua

```diff
@@ -438,6 +440,25 @@ function osm2pgsql.process_node(object)
     if object.tags.place and object.tags.wikidata == 'Q200667' then
         output_hstore['disputed_by'] = 'PK'
     end
+-- Recast Gaza Strip label as country
+    if object.tags.place and object.tags.wikidata == 'Q39760' then
+        output_hstore['place'] = 'country'
+        output_hstore['disputed_by'] = 'SA;PK;ID'
```

Comment:
But then this turns it off?

Reply:
Yeah, its place=region in the data. So I upgrade it to place=country then add dispute_by to mark it as kind:xx = unrecognized to those countries.